pkg/plugins/datasource_plugin.go

```diff
@@ -19,6 +19,7 @@ type DataSourcePlugin struct {
 	FrontendPluginBase
 	Annotations  bool              `json:"annotations"`
 	Metrics      bool              `json:"metrics"`
+	Unified      bool              `json:"unified"`
```

Comment:
what is unified?  and does it make sense to even have these on the datasource plugin?  

I don't know of any datasource that should be only metrics or only logs -- even loki can be metrics!

Reply:
Unified identifies a datasource as having a unified interface in Explore such that a user doesn't need to toggle between Logs and Metrics modes depending on the type of query.

Reply:
We discussed this just now and we'll look for a different path forward that won't need this property.